crates/autopilot/src/arguments.rs

```diff
@@ -203,6 +207,16 @@ pub struct Arguments {
         value_parser = shared::arguments::duration_from_seconds,
     )]
     pub solve_deadline: Duration,
+
+    /// Time interval in days between each cleanup operation of the
+    /// `order_events` database table.
+    #[clap(long, env, default_value = "1", value_parser = duration_from_days)]
```

Comment:
TBD: default value

Reply:
I'm even more positive on your suggested `humanreadtime` change, as I find it dangerous to configure different time parameters in different units (given the name doesn't indicate in which unit this value is expected to be taken)

Reply:
Will be done in a separate PR https://github.com/cowprotocol/services/issues/2085